### PR TITLE
docs: Update docs in CONTRIBUTING and test commands

### PR DESCRIPTION
**Pull request by thebrianchen**

### CODE REVIEW

File: .kokoro/samples-test.sh [hunk 1]
```diff
@@ -40,7 +40,7 @@ if [ -f samples/package.json ]; then
     npm install
     cd ..
 
-    npm run samples-test
+    npm run test:samples
```
Comment (JustinBeckwith) [inline .kokoro/samples-test.sh]: These files are auto-generated, and we put a *huge* premium on consistency here.

Comment (schmidt-sebastian) [reply .kokoro/samples-test.sh]: We tried to make this SDK more consistent with the test patterns we use in `firebase-js-sdk`. That being said, breaking (or worse: fixing) 72 repos was not our goal :) 

If I am not mistaken, then this repo is the only repo that uses the conformance tests. We should be able to achieve some of the goals of this PR (have more consistency) by just updating the conformance test runner to 'conformance-test' or by just inlining it with the unit tests.

**Comment by schmidt-sebastian** [comment]
Further things we should mention:
- Docs generation and docs testing
- Running the system tests and how to set up the environment variable

**Comment by schmidt-sebastian** [comment]
@thebrianchen Should we close this for now? https://github.com/googleapis/nodejs-firestore/pull/780 merges your latest changes.

**Pull request closed without merging**